.ci/update-tekton-bundle.sh

```diff
@@ -0,0 +1,69 @@
+#!/bin/bash
+TEMP_FOLDER=$WORKSPACE/tmp/bundle-override
+APPSTUDIO_QE_REPO=quay.io/redhat-appstudio-qe/test-images
+TASK_BUNDLE_IMG=$APPSTUDIO_QE_REPO:task-bundle-$JVM_BUILD_SERVICE_PR_SHA
+export PIPELINE_BUNDLE_IMG=$APPSTUDIO_QE_REPO:pipeline-bundle-$JVM_BUILD_SERVICE_PR_SHA
+
+function setTargetBundle() {
+    TARGET_BUNDLE=$(yq e ".configMapGenerator[].literals[]" \
+    $WORKSPACE/tmp/infra-deployments/components/build/kustomization.yaml \
+    | grep default_build_bundle \
+    | sed "s/default_build_bundle=//")
+
+    echo New build bundle will be created based on $TARGET_BUNDLE
+}
+
+function createPipelineFile() {
+    tkn bundle list -o yaml $TARGET_BUNDLE pipeline java-builder > $TEMP_FOLDER/pipeline.yaml
```

Comment:
I only fetched a single pipeline here because when `tkn` outputs the list of pipelines, it does not create a break between them, so this affects how yq parses it.